content-src/components/LinkMenu/LinkMenu.js

```diff
@@ -38,9 +38,16 @@ const LinkMenu = React.createClass({
           label: "Dismiss",
           icon: "dismiss",
           userEvent: "BLOCK",
-          onClick: () => dispatch(actions.NotifyBlockURL(site.url))
+          onClick: () => {
+            if (site.recommended) {
+              dispatch(actions.NotifyBlockRecommendation(site.url));
+              dispatch(actions.RequestHighlightsLinks());
```

Comment:
is it possible to waitUntil ```NotifyBlockRecommendation``` finishes and then that function can make a call to ```RequestHighlightsLinks``` when it's done?